tools/dendrite_openapi_version

```diff
@@ -1,2 +1,2 @@
-COMMIT="be23ee5add60504a389156a9ae54f6a6831adf34"
-SHA2="f425a068858ac4525272ba281f6ff31f784f1a48ca72fed1dad92fda25bbc94a"
+COMMIT="eb7d819804331b64bf5588c85e799e0f30306f8a"
```

Comment:
Not a huge deal, but might want to move this up to the latest `main` since it's changing anyway.